# Czech translation for duplicity
# Copyright (c) 2014 Rosetta Contributors and Canonical Ltd 2014
# This file is distributed under the same license as the duplicity package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2020-03-07 09:49-0600\n"
"PO-Revision-Date: 2020-03-13 14:43+0000\n"
"Last-Translator: Pavel Borecki <Unknown>\n"
"Language-Team: Czech <cs@li.org>\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2020-03-18 06:31+0000\n"
"X-Generator: Launchpad (build 3a6db24bbe7280ec09bae73384238390fcc98ad3)\n"

#: ../bin/duplicity:120
msgid "INT intercepted...exiting."
msgstr "Zachyceno přerušení, ukončuje se."

#: ../bin/duplicity:128
#, python-format
msgid "GPG error detail: %s"
msgstr "Podrobnosti chyby GPG: %s"

#: ../bin/duplicity:138
#, python-format
msgid "User error detail: %s"
msgstr "Podrobnosti uživateské chyby: %s"

#: ../bin/duplicity:148
#, python-format
msgid "Backend error detail: %s"
msgstr "Podrobnosti chyby na podpůrné vrstvě (backend): %s"

#: ../bin/rdiffdir:64 ../duplicity/commandline.py:236
#, python-format
msgid "Error opening file %s"
msgstr "Chyba při otevírání souboru %s"

#: ../bin/rdiffdir:131
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr "Soubor %s už existuje, nebude přepsán."

#: ../duplicity/asyncscheduler.py:70
#, python-format
msgid "instantiating at concurrency %d"
msgstr "doloženo příkladem v konkurenci %d"

#: ../duplicity/asyncscheduler.py:97
msgid "inserting barrier"
msgstr "vkládání bariéry"

#: ../duplicity/asyncscheduler.py:146
msgid "running task synchronously (asynchronicity disabled)"
msgstr "spouštění úlohy synchronně (asynchronní je vypnuté)"

#: ../duplicity/asyncscheduler.py:152
msgid "scheduling task for asynchronous execution"
msgstr "naplánování úlohy pro asynchronní vykonání"

#: ../duplicity/asyncscheduler.py:181
msgid "task completed successfully"
msgstr "úloha úspěšně dokončena"

#: ../duplicity/asyncscheduler.py:192
msgid ""
"a previously scheduled task has failed; propagating the result immediately"
msgstr ""
"dříve naplánovaná úloha se nezdařila, výsledek je proto propagován okamžitě"

#: ../duplicity/asyncscheduler.py:215 ../duplicity/asyncscheduler.py:236
#, python-format
msgid "active workers = %d"
msgstr "aktivní zpracovávající procesy = %d"

#: ../duplicity/asyncscheduler.py:256
#, python-format
msgid "task execution done (success: %s)"
msgstr "vykonání úlohy dokončeno (úspěch: %s)"

#: ../duplicity/backend.py:104
#, python-format
msgid "Import of %s %s"
msgstr "Import %s %s"

#: ../duplicity/backend.py:213
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Nedaří se inicializovat podpůrnou vrstvu: %s"

#: ../duplicity/backend.py:392
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Výpis zpětného volání funkce (backtrace) u předchozí chyby: %s"

#: ../duplicity/backend.py:408
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Vzdává se po %s pokusech. %s: %s"

#: ../duplicity/backend.py:412
#, python-format
msgid "Attempt %s failed. %s: %s"
msgstr "Pokus %s se nezdařil. %s: %s"

#: ../duplicity/backend.py:506
#, python-format
msgid "Reading results of '%s'"
msgstr "Čtení výsledků „%s“"

#: ../duplicity/backend.py:532
#, python-format
msgid "Writing %s"
msgstr "Zápis %s"

#: ../duplicity/backend.py:573
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Soubor %s nebyl nalezen lokálně po získání z podpůrné vrstvy (backend)"

#: ../duplicity/backends/giobackend.py:109
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Připojení se nezdařilo, prosím zkontrolujte své heslo: %s"

#: ../duplicity/backends/multibackend.py:97
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend: nepodařilo se zpracovat řetězec dotazu %s: %s "

#: ../duplicity/backends/multibackend.py:106
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr ""
"MultiBackend: neplatný řetězec dotazu %s: více než jedna hodnota pro %s"

#: ../duplicity/backends/multibackend.py:111
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultiBackend: neplatný řetězec dotazu %s: neznámý parametr %s"

#: ../duplicity/backends/multibackend.py:161
#: ../duplicity/backends/multibackend.py:166
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultiBackend: neplatná hodnota pro %s: %s"

#: ../duplicity/backends/multibackend.py:177
#, python-format
msgid "MultiBackend: Url %s"
msgstr "MultiBackend: URL %s"

#: ../duplicity/backends/multibackend.py:181
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultiBackend: Nedaří se načíst soubor s nastavením %s: %s "

#: ../duplicity/backends/multibackend.py:190
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend: použít uložení %s"

#: ../duplicity/backends/multibackend.py:195
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "MultiBackend: nastavit proměnnou prostředí %s = %s"

#: ../duplicity/backends/multibackend.py:208
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "Multibackend: registrovat přiřazení pro předponu %s"

#: ../duplicity/backends/multibackend.py:249
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MultiBackend: _put: zapsat do uložení č. %s (%s)"

#: ../duplicity/backends/multibackend.py:262
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr ""
"MultiBackend: nepodařilo se zapsat do uložení č. %s (%s), zkuste č. %s, "
"výjimka: %s"

#: ../duplicity/backends/multibackend.py:269
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MultiBackend: nezdařilo se zapsat do %s. Proces bude přerušen."

#: ../duplicity/backends/multibackend.py:276
#, python-format
msgid ""
"MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr ""
"MultiBackend: nepodařilo se zapsat %s. Vyzkoušena všechny uložení a v žádném "
"se neuspělo"

#: ../duplicity/backends/multibackend.py:295
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MultiBackend: nepodařilo se získat %s do %s z %s"

#: ../duplicity/backends/multibackend.py:298
#, python-format
msgid ""
"MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr ""
"MultiBackend: nepodařilo se získat %s. Vyzkoušena všechna úložiště a na "
"žádném neuspělo"

#: ../duplicity/backends/multibackend.py:308
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "MultiBackend: %s: %d souborů"

#: ../duplicity/backends/multibackend.py:311
#, python-format
msgid "Exception during list of %s: %s"
msgstr "Výjimka při vypisování %s: %s"

#: ../duplicity/backends/multibackend.py:318
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MultiBackend: kombinovaný seznam: %s"

#: ../duplicity/backends/multibackend.py:346
#, python-format
msgid "MultiBackend: failed to delete %s from %s"
msgstr "MultiBackend: nepodařilo se smazat %s z %s"

#: ../duplicity/backends/multibackend.py:350
#, python-format
msgid ""
"MultiBackend: failed to delete %s. Tried all backing stores and none "
"succeeded"
msgstr ""
"MultiBackend: nepodařilo se smazat %s. Vyzkoušena všechna úložiště a na "
"žádném se neuspělo"

#: ../duplicity/backends/pydrivebackend.py:150
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "Podpůrná vrstva PyDrive: vícero souborů nazvaných „%s“."

#: ../duplicity/backends/webdavbackend.py:66
msgid "Missing socket or ssl python modules."
msgstr "Chybějící python moduly pro socket nebo ssl."

#: ../duplicity/backends/webdavbackend.py:84
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Soubor „%s“ s certifikáty cert. autorit není čitelný."

#: ../duplicity/backends/webdavbackend.py:105
msgid ""
"Option '--ssl-cacert-path' is not supported with python 2.7.8 and below."
msgstr "Volba „--ssl-cacert-path“ není podporovaná v python 2.7.8 a starších."

#: ../duplicity/backends/webdavbackend.py:109
#, python-format
msgid ""
"For certificate verification with python 2.7.8 or earlier a cacert database\n"
"file is needed in one of these locations: %s\n"
"Hints:\n"
"Consult the man page, chapter 'SSL Certificate Verification'.\n"
"Consider using the options --ssl-cacert-file, --ssl-no-check-certificate ."
msgstr ""
"Pro ověřování certifikátů s python 2.7.8 a starším je zapotřebí "
"databázového\n"
"souboru cacert v jednom z těchto umístění: %s\n"
"Rady:\n"
"Konzultujte s manuálovou stránkou, kapitolou „SSL Certificate "
"Verification“.\n"
"Zvažte použití voleb --ssl-cacert-file, --ssl-no-check-certificate ."

#: ../duplicity/backends/webdavbackend.py:155
#, python-format
msgid "Using WebDAV protocol %s"
msgstr "Pomocí WebDAV protokolu %s"

#: ../duplicity/backends/webdavbackend.py:156
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "S použitím WebDAV hostitele %s, port %s"

#: ../duplicity/backends/webdavbackend.py:158
#, python-format
msgid "Using WebDAV directory %s"
msgstr "S použitím WebDAV složky %s"

#: ../duplicity/backends/webdavbackend.py:189
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "WebDAV vytvořit spojení na „%s“"

#: ../duplicity/backends/webdavbackend.py:200
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "WebDAV neznámé schéma URI: %s"

#: ../duplicity/backends/webdavbackend.py:219
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s požadavek s hlavičkami: %s "

#: ../duplicity/backends/webdavbackend.py:220
#: ../duplicity/backends/webdavbackend.py:243
#, python-format
msgid "WebDAV data length: %s "
msgstr "WebDAV datový objem: %s "

#: ../duplicity/backends/webdavbackend.py:223
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "Stav %s WebDAV odpovědi s důvodem „%s“."

#: ../duplicity/backends/webdavbackend.py:229
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "WebDAV přeměrovat na: %s "

#: ../duplicity/backends/webdavbackend.py:231
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV přesměroval 10 krát. Dále už se zkoušet nebude."

#: ../duplicity/backends/webdavbackend.py:236
msgid "WebDAV missing location header in redirect response."
msgstr "WebDAV chybí hlavička umístění v přesměrovávací odpovědi."

#: ../duplicity/backends/webdavbackend.py:241
msgid "WebDAV retry request with authentification headers."
msgstr "WebDAV požadavek na opětovný pokus s ověřovacími hlavičkami."

#: ../duplicity/backends/webdavbackend.py:242
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s druhý požadavek s hlavičkami: %s "

#: ../duplicity/backends/webdavbackend.py:246
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "WebDAV stav %s druhé odpovědi s důvodem „%s“."

#: ../duplicity/backends/webdavbackend.py:263
msgid ""
"python-kerberos needed to use kerberos                           "
"authorization, falling back to basic auth."
msgstr ""
"Pro použití kerberos ověřování je třeba python-kerberos, náhradně se použije "
"ověření uživatelským jménem a heslem."

#: ../duplicity/backends/webdavbackend.py:267
#, python-format
msgid ""
"Kerberos authorization failed: %s.                          Falling back to "
"basic auth."
msgstr ""
"Kerberos ověřování se nezdařilo: %s. Náhradně se bude ověřovat uživatelským "
"jménem a heslem."

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "Vytváření chybějící složky %s"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s se nezdařilo: %s %s"

#: ../duplicity/backends/webdavbackend.py:382
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "WebDAV dekódování a překlad popisu umístění: %s -> %s"

#: ../duplicity/backends/webdavbackend.py:427
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "WebDAV GET chybný stavový kód %s důvod %s."

#: ../duplicity/backends/webdavbackend.py:449
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT chybný stavový kód %s důvod %s."

#: ../duplicity/backends/webdavbackend.py:469
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL chybný stavový kód %s důvod %s."

#: ../duplicity/commandline.py:79
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"Use of default filenames is strongly suggested."
msgstr ""
"Varování: Volbu %s čeká zastarání a bude odebrána v budoucím vydání.\n"
"Použití výchozích názvů souborů je silně doporučováno."

#: ../duplicity/commandline.py:86
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"--include-filelist and --exclude-filelist now accept globbing characters and "
"should be used instead."
msgstr ""
"Varování: Volbu %s čeká zastarání a bude odebrána v budoucím vydání.\n"
"volby --include-filelist a --exclude-filelist nyní přijímají zástupné znaky "
"a měly by být použity namísto toho."

#: ../duplicity/commandline.py:96
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"On many GNU/Linux systems, stdin is represented by /dev/stdin and\n"
"--include-filelist=/dev/stdin or --exclude-filelist=/dev/stdin could\n"
"be used as a substitute."
msgstr ""
"Varování: Volbu %s čeká zastarání a bude odebrána v budoucím vydání.\n"
"Na mnoha GNU/Linux systémech, stdin je představován /dev/stdin a\n"
"volby --include-filelist=/dev/stdin nebo --exclude-filelist=/dev/stdin "
"mohou\n"
"být použity jako náhrada."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:257 ../duplicity/commandline.py:267
#: ../duplicity/commandline.py:288 ../duplicity/commandline.py:359
#: ../duplicity/commandline.py:377 ../duplicity/commandline.py:609
#: ../duplicity/commandline.py:642 ../duplicity/commandline.py:855
msgid "path"
msgstr "popis umístění"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:283 ../duplicity/commandline.py:290
#: ../duplicity/commandline.py:383 ../duplicity/commandline.py:593
#: ../duplicity/commandline.py:828
msgid "gpg-key-id"
msgstr "identifikator_gpg_klice"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/commandline.py:298 ../duplicity/commandline.py:408
#: ../duplicity/commandline.py:878
msgid "shell_pattern"
msgstr "shellovy_vzor"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:304 ../duplicity/commandline.py:313
#: ../duplicity/commandline.py:320 ../duplicity/commandline.py:410
#: ../duplicity/commandline.py:417 ../duplicity/commandline.py:430
#: ../duplicity/commandline.py:824
msgid "filename"
msgstr "soubor"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:327 ../duplicity/commandline.py:421
msgid "regular_expression"
msgstr "r_egulární výraz"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:331 ../duplicity/commandline.py:371
#: ../duplicity/commandline.py:497 ../duplicity/commandline.py:910
msgid "time"
msgstr "čas"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:379 ../duplicity/commandline.py:477
#: ../duplicity/commandline.py:500 ../duplicity/commandline.py:601
#: ../duplicity/commandline.py:843
msgid "options"
msgstr "volby"

#: ../duplicity/commandline.py:394
#, python-format
msgid ""
"Running in 'ignore errors' mode due to %s; please re-consider if this was "
"not intended"
msgstr ""
"Spuštěno v režimu „ignorovat chyby“ kvůli %s – znovu zvažte zda toto bylo "
"zamýšleno"

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:406
msgid "imap_mailbox"
msgstr "imap_schranka"

#: ../duplicity/commandline.py:424
msgid "file_descriptor"
msgstr "popisovač souboru"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:435 ../duplicity/commandline.py:457
#: ../duplicity/commandline.py:474 ../duplicity/commandline.py:483
#: ../duplicity/commandline.py:533 ../duplicity/commandline.py:538
#: ../duplicity/commandline.py:542 ../duplicity/commandline.py:561
#: ../duplicity/commandline.py:567 ../duplicity/commandline.py:571
#: ../duplicity/commandline.py:637 ../duplicity/commandline.py:838
msgid "number"
msgstr "číslo"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:438
msgid "backup name"
msgstr "název zálohy"

#: ../duplicity/commandline.py:556
msgid "policy"
msgstr "zásada"

#: ../duplicity/commandline.py:574
msgid "Hot|Cool|Archive"
msgstr ""

#. noun
#: ../duplicity/commandline.py:577 ../duplicity/commandline.py:580
#: ../duplicity/commandline.py:809
msgid "command"
msgstr "příkaz"

#: ../duplicity/commandline.py:583
msgid "pyrax|cloudfiles"
msgstr "pyrax|cloudfiles"

#: ../duplicity/commandline.py:604
msgid "pem formatted bundle of certificate authorities"
msgstr "balíček certifikátů cert. autorit ve formátu pem"

#: ../duplicity/commandline.py:605
msgid "path to a folder with certificate authority files"
msgstr "popis umístění složky se soubory certifikačních autorit"

#. Used in usage help. Example:
#. --timeout <seconds>
#. Used in usage help. Example:
#. --backend-retry-delay <seconds>
#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:614 ../duplicity/commandline.py:648
#: ../duplicity/commandline.py:872
msgid "seconds"
msgstr "sekund"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:620 ../duplicity/commandline.py:806
msgid "char"
msgstr "znak"

#: ../duplicity/commandline.py:772
#, python-format
msgid "Using archive dir: %s"
msgstr "S použitím archivní složky: %s"

#: ../duplicity/commandline.py:773
#, python-format
msgid "Using backup name: %s"
msgstr "S použitím názvu zálohy: %s"

#: ../duplicity/commandline.py:780
#, python-format
msgid "Command line error: %s"
msgstr "Chyba na příkazovém řádku: %s"

#: ../duplicity/commandline.py:781
msgid "Enter 'duplicity --help' for help screen."
msgstr "Pro nápovědu zadejte „duplicty --help“."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:794
msgid "absolute_path"
msgstr "absolutni_popis_umisteni"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:798
msgid "alias"
msgstr "alternativni_nazev"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:803
msgid "bucket_name"
msgstr "nazev_kontejneru"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:814
msgid "container_name"
msgstr "název_kontejneru"

#. noun
#: ../duplicity/commandline.py:817
msgid "count"
msgstr "počet"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:820
msgid "directory"
msgstr "složka"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:833
msgid "module"
msgstr "modul"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:847
msgid "other.host"
msgstr "druhy.stroj"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:851
msgid "password"
msgstr "heslo"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:859
msgid "port"
msgstr "port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:864
msgid "prefix"
msgstr "předpona"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:868
msgid "relative_path"
msgstr "relativni_popis_umisteni"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:883
msgid "some_dir"
msgstr "nejaka_slozka"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:889
msgid "source_dir"
msgstr "zdrojová_složka"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:894
msgid "source_url"
msgstr "zdrojova_url"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:900
msgid "target_dir"
msgstr "cílová_složka"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:905
msgid "target_url"
msgstr "cilova_url"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:915
msgid "user"
msgstr "uživatel"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/commandline.py:918
msgid "account_id"
msgstr "identifikator_uctu"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/commandline.py:922
msgid "application_key"
msgstr "aplikacni_klic"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/commandline.py:926 ../duplicity/dup_collections.py:511
msgid "remote"
msgstr "vzdálená"

#. Header in usage help
#: ../duplicity/commandline.py:945
msgid "Backends and their URL formats:"
msgstr "Podpůrné vrstvy (backend) a formáty jejich URL adres:"

#. Header in usage help
#: ../duplicity/commandline.py:978
msgid "Commands:"
msgstr "Příkazy:"

#: ../duplicity/commandline.py:1003
#, python-format
msgid "Specified archive directory '%s' does not exist, or is not a directory"
msgstr "Zadaná složka archivu „%s“ neexistuje, nebo se nejedná o složku"

#: ../duplicity/commandline.py:1012
#, python-format
msgid ""
"Sign key should be an 8, 16 alt. 40 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr ""
"Podpisový klíč by měl být 8, 16 a případně 40 znaků dlouhý hex řetězec jako "
"např. „AA0E73D2“.\n"
"Namísto toho obdrženo „%s“."

#: ../duplicity/commandline.py:1072
#, python-format
msgid ""
"Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr ""
"Složka %s do které se má obnovit už existuje.\n"
"Nebude přepsána."

#: ../duplicity/commandline.py:1077
#, python-format
msgid "Verify directory %s does not exist"
msgstr "Ověřovací složka %s neexistuje"

#: ../duplicity/commandline.py:1083
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "Zdrojová složka zálohy %s neexistuje."

#: ../duplicity/commandline.py:1114
#, python-format
msgid "Command line warning: %s"
msgstr "Varování příkazového řádku: %s"

#: ../duplicity/commandline.py:1114
msgid ""
"Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr ""
"Volby výběru --exclude/--include\n"
"v tuto chvíli fungují pouze pro zálohování, ne při obnovování."

#: ../duplicity/commandline.py:1150
#, python-format
msgid "GPG binary is %s, version %s"
msgstr "Spustitelný soubor GPG je %s, verze %s"

#: ../duplicity/commandline.py:1177
#, python-format
msgid ""
"Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr ""
"Chybná URL „%s“.\n"
"Příklady URL řetězců jsou „scp://uzivatel@stroj.net:1234/popis_umisteni“ a\n"
"\"file:///usr/local“. Další informace viz manuálová stránka."

#: ../duplicity/commandline.py:1207
msgid "Main action: "
msgstr "Hlavní akce: "

#: ../duplicity/diffdir.py:115 ../duplicity/diffdir.py:418
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Chyba %s při získávání rozdílu pro %s"

#: ../duplicity/diffdir.py:130
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Získávání rozdílu %s a %s"

#: ../duplicity/diffdir.py:181
#, python-format
msgid "A %s"
msgstr "A %s"

#: ../duplicity/diffdir.py:188
#, python-format
msgid "M %s"
msgstr "M %s"

#: ../duplicity/diffdir.py:210
#, python-format
msgid "Comparing %s and %s"
msgstr "Porovnávání %s a %s"

#: ../duplicity/diffdir.py:218
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/dup_collections.py:180 ../duplicity/dup_collections.py:194
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete: chybějící %s"

#: ../duplicity/dup_collections.py:219
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Fatální chyba: nebyly nalezeny žádné manifesty pro nejnovější zálohu"

#: ../duplicity/dup_collections.py:228
msgid ""
"Fatal Error: Remote manifest does not match local one.  Either the remote "
"backup set or the local archive directory has been corrupted."
msgstr ""
"Fatální chyba: Vzdálený manifest se neshoduje s tím místním. Buď byla "
"poškozena vzdálená sada záloh, nebo místní archivní složka."

#: ../duplicity/dup_collections.py:236
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Fatální chyba: Ani vzdálený, ani místní manifest nejsou čitelné."

#: ../duplicity/dup_collections.py:247
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Zpracovávání místního manifestu %s (%s)"

#: ../duplicity/dup_collections.py:259
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Chyba při zpracovávání místního manifestu (%s): %s"

#: ../duplicity/dup_collections.py:262
#, python-format
msgid "Processing remote manifest %s (%s)"
msgstr "Zpracovávání vzdáleného manifestu %s (%s)"

#: ../duplicity/dup_collections.py:358
msgid "Preferring Backupset over previous one!"
msgstr "Upřednostňování sady záloh před předchozím!"

#: ../duplicity/dup_collections.py:361
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Ignoruje se přírůstková sada záloh (start_time: %s, potřebný: %s)"

#: ../duplicity/dup_collections.py:366
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Přidána přírůstková sada záloh (start_time: %s / end_time: %s)"

#: ../duplicity/dup_collections.py:436
msgid "Chain start time: "
msgstr "Čas začátku řetězu: "

#: ../duplicity/dup_collections.py:437
msgid "Chain end time: "
msgstr "Čas konce řetězu: "

#: ../duplicity/dup_collections.py:438
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Počet obsažených sad záloh: %d"

#: ../duplicity/dup_collections.py:440
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Celkový počet obsažených svazků: %d"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Type of backup set:"
msgstr "Typ sad záloh:"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Time:"
msgstr "Čas:"

#: ../duplicity/dup_collections.py:442
msgid "Num volumes:"
msgstr "Poč. svazků:"

#: ../duplicity/dup_collections.py:446 ../duplicity/dup_collections.py:1237
msgid "Full"
msgstr "Plná"

#: ../duplicity/dup_collections.py:449 ../duplicity/dup_collections.py:1239
msgid "Incremental"
msgstr "Přírůstková"

#: ../duplicity/dup_collections.py:509
msgid "local"
msgstr "místní"

#: ../duplicity/dup_collections.py:666
msgid "Collection Status"
msgstr "Stav sběru"

#: ../duplicity/dup_collections.py:668
#, python-format
msgid "Connecting with backend: %s"
msgstr "Spojování se s podpůrnou vrstvou: %s"

#: ../duplicity/dup_collections.py:670
#, python-format
msgid "Archive dir: %s"
msgstr "Archivní složka: %s"

#: ../duplicity/dup_collections.py:673
#, python-format
msgid "Found %d secondary backup chain."
msgid_plural "Found %d secondary backup chains."
msgstr[0] "Nalezen %d pomocný řetězec zálohy."
msgstr[1] "Nalezeny %d pomocné řetězce zálohy."
msgstr[2] "Nalezeno %d pomocných řetězců zálohy."

#: ../duplicity/dup_collections.py:678
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Sekundární řetěz %d z %d:"

#: ../duplicity/dup_collections.py:684
msgid "Found primary backup chain with matching signature chain:"
msgstr "Nalezen hlavní řetěz zálohy s odpovídajícím podpisovým řetězem:"

#: ../duplicity/dup_collections.py:688
msgid "No backup chains with active signatures found"
msgstr "Nebyly nalezený žádné řetězce záloh s aktivními podpisy"

#: ../duplicity/dup_collections.py:691
#, python-format
msgid "Also found %d backup set not part of any chain,"
msgid_plural "Also found %d backup sets not part of any chain,"
msgstr[0] "Také nalezena %d sada zálohy která není součástí žádného řetězce,"
msgstr[1] ""
"Také nalezeny %d sady zálohy, které nejsou součástí žádného řetězce,"
msgstr[2] "Také nalezeno %d sad záloh, které nejsou součástí žádného řetězce,"

#: ../duplicity/dup_collections.py:695
#, python-format
msgid "and %d incomplete backup set."
msgid_plural "and %d incomplete backup sets."
msgstr[0] "a %d neúplná sada záloh."
msgstr[1] "a %d neúplné sady záloh."
msgstr[2] "a %d neúplných sad záloh."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:700
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr ""
"Ty mohou být smazány spuštěním duplicity s příkazem „cleanup“ (vyčištění)."

#: ../duplicity/dup_collections.py:703
msgid "No orphaned or incomplete backup sets found."
msgstr "Nebyly nalezeny žádné k ničemu nepatřící nebo neúplné sady záloh."

#: ../duplicity/dup_collections.py:719
#, python-format
msgid "%d file exists on backend"
msgid_plural "%d files exist on backend"
msgstr[0] "%d soubor existuje na podpůrné vrstvě"
msgstr[1] "%d soubory existují na podpůrné vrstvě"
msgstr[2] "%d souborů existuje na podpůrné vrstvě"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] "%d soubor existuje v mezipaměti"
msgstr[1] "%d soubory existují v mezipaměti"
msgstr[2] "%d souborů existuje v mezipaměti"

#: ../duplicity/dup_collections.py:782
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr ""
"Varování, zahazuje se nejnovější sada záloh, protože chybí soubor s podpisem."

#: ../duplicity/dup_collections.py:805
msgid "Warning, found the following local orphaned signature file:"
msgid_plural "Warning, found the following local orphaned signature files:"
msgstr[0] ""
"Varování, nalezen následující místní, k ničemu nepatřící podpisový soubor:"
msgstr[1] ""
"Varování, nalezeny následující místní, k ničemu nepatřící podpisové soubory:"
msgstr[2] ""
"Varování, nalezeno následujících místních, k ničemu nepatřící podpisových "
"souborů:"

#: ../duplicity/dup_collections.py:814
msgid "Warning, found the following remote orphaned signature file:"
msgid_plural "Warning, found the following remote orphaned signature files:"
msgstr[0] ""
"Varování, nalezen následující vzdálený soubor s podpisem, který k ničemu "
"nepatří:"
msgstr[1] ""
"Varování, nalezeny následující vzdálené soubory s podpisem, které k ničemu "
"nepatří:"
msgstr[2] ""
"Varování, nalezeno následujících vzdálených souborů s podpisem, které k "
"ničemu nepatří:"

#: ../duplicity/dup_collections.py:823
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Varování, nalezeny podpisy ale žádné odpovídající soubory záloh"

#: ../duplicity/dup_collections.py:827
msgid ""
"Warning, found incomplete backup sets, probably left from aborted session"
msgstr ""
"Varování, nalezeny neúplné sady záloh, pravděpodobně zůstaly pro přerušeném "
"sezení"

#: ../duplicity/dup_collections.py:831
msgid "Warning, found the following orphaned backup file:"
msgid_plural "Warning, found the following orphaned backup files:"
msgstr[0] ""
"Varování, nalezen následující soubor zálohy, který k ničemu nepatří:"
msgstr[1] ""
"Varování, nalezeny následující soubory zálohy, které k ničemu nepatří:"
msgstr[2] ""
"Varování, nalezeno následujících souborů zálohy, které k ničemu nepatří:"

#: ../duplicity/dup_collections.py:848
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Vyzískávají se řetězce zálohy ze seznamu souborů: %s"

#: ../duplicity/dup_collections.py:860
#, python-format
msgid "File %s is part of known set"
msgstr "Soubor %s je součástí známé sady"

#: ../duplicity/dup_collections.py:863
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Soubor %s není součástí známé sady, bude proto vytvořena nová"

#: ../duplicity/dup_collections.py:868
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Ignoruje se soubor (odmítnutý sadou záloh) „%s“"

#: ../duplicity/dup_collections.py:884
#, python-format
msgid "Found backup chain %s"
msgstr "Nalezen řetězec záloh %s"

#: ../duplicity/dup_collections.py:889
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Přidána sada %s k už existujícímu řetězci %s"

#: ../duplicity/dup_collections.py:893
#, python-format
msgid "Found orphaned set %s"
msgstr "Nalezena sada %s, která k ničemu nepatří"

#: ../duplicity/dup_collections.py:1043
#, python-format
msgid ""
"No signature chain for the requested time. Using oldest available chain, "
"starting at time %s."
msgstr ""
"Pro požadovaný čas neexistuje žádný řetězec podpisu. Náhradně se použije "
"starší dostupný řetězec, začínající v čase %s."

#: ../duplicity/dup_collections.py:1229
#, python-format
msgid "File: %s"
msgstr "Soubor: %s"

#: ../duplicity/dup_collections.py:1230
#, python-format
msgid "Total number of backup: %d"
msgstr "Celkový počet záloh: %d"

#: ../duplicity/dup_collections.py:1231
msgid "Type of file change:"
msgstr "Typ změny souboru:"

#: ../duplicity/dup_main.py:121
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Využít nastavenou PASSPHRASE i jako SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:128
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Využít nastavenou SIGN_PASSPHRASE i jako PASSPHRASE"

#: ../duplicity/dup_main.py:167
msgid "PASSPHRASE variable not set, asking user."
msgstr "Proměnná PASSPHRASE není nastavená, dotazuje se uživatele."

#: ../duplicity/dup_main.py:182
msgid "GnuPG passphrase for signing key:"
msgstr "GnuPG heslová fráze pro podpisový klíč:"

#: ../duplicity/dup_main.py:187
msgid "GnuPG passphrase:"
msgstr "GnuPG heslová fráze:"

#: ../duplicity/dup_main.py:192
msgid "Retype passphrase for signing key to confirm: "
msgstr "Znovu zadejte heslovou frázi pro podpisový klíč (pro potvrzení): "

#: ../duplicity/dup_main.py:194
msgid "Retype passphrase to confirm: "
msgstr "Znovu zadejte heslovou frázi pro potvrzení: "

#: ../duplicity/dup_main.py:197
msgid "First and second passphrases do not match!  Please try again."
msgstr "První a druhá heslová fráze se neshodují! Zkuste to znovu."

#: ../duplicity/dup_main.py:204
msgid ""
"Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr ""
"Při symetrickém šifrování není možné nevyplnit heslovou frázi! Zkuste to "
"znovu."

#: ../duplicity/dup_main.py:261
#, python-format
msgid ""
"File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Soubor %s dokončen v sadě zálohy.\n"
"Pokračuje se restartem na souboru %s."

#: ../duplicity/dup_main.py:270
#, python-format
msgid ""
"File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Soubor %s chybí v sadě zálohy.\n"
"Pokračuje se restartem na souboru %s."

#: ../duplicity/dup_main.py:319
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Soubor %s byl během nahrávání poškozen."

#: ../duplicity/dup_main.py:359
msgid ""
"Restarting backup, but current encryption settings do not match original "
"settings"
msgstr ""
"Záloha se restartuje, ale stávající nastavení šifrování se neshodují s "
"původními nastaveními"

#: ../duplicity/dup_main.py:382
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Restartuje se po svazku %s, soubor %s, blok %s"

#: ../duplicity/dup_main.py:452
#, python-format
msgid "Processed volume %d"
msgstr "Zpracovaný svazek %d"

#: ../duplicity/dup_main.py:602
msgid ""
"Fatal Error: Unable to start incremental backup.  Old signatures not found "
"and incremental specified"
msgstr ""
"Fatální chyba: Nedaří se spustit přírůstkovou zálohu. Staré podpisy nebyly "
"nalezeny a přírůstková určena"

#: ../duplicity/dup_main.py:606
msgid "No signatures found, switching to full backup."
msgstr "Nebyly nalezeny podpisy, přepíná se na plnou zálohu."

#: ../duplicity/dup_main.py:620
msgid "Backup Statistics"
msgstr "Statistiky záloh"

#: ../duplicity/dup_main.py:726
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s nenalezeno v archivu – nebyly obnoveny žádné soubory."

#: ../duplicity/dup_main.py:730
msgid "No files found in archive - nothing restored."
msgstr "V archivu nejsou žádné soubory - nic nebylo obnoveno."

#: ../duplicity/dup_main.py:763
#, python-format
msgid "Processed volume %d of %d"
msgstr "Zpracován svazek %d z %d"

#: ../duplicity/dup_main.py:797
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Neplatná data - nesouhlasí otisk (hash) %s pro soubor:"

#: ../duplicity/dup_main.py:800
#, python-format
msgid "Calculated hash: %s"
msgstr "Spočítaný otisk (hash): %s"

#: ../duplicity/dup_main.py:801
#, python-format
msgid "Manifest hash: %s"
msgstr "Otisk z manifestu: %s"

#: ../duplicity/dup_main.py:844
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Svazek byl podpesán klíčem %s, nikoli %s"

#: ../duplicity/dup_main.py:876
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Kontrola dokončena: %s, %s."

#: ../duplicity/dup_main.py:877
#, python-format
msgid "%d file compared"
msgid_plural "%d files compared"
msgstr[0] "%d soubor porovnán"
msgstr[1] "%d soubory porovnány"
msgstr[2] "%d souborů porovnáno"

#: ../duplicity/dup_main.py:879
#, python-format
msgid "%d difference found"
msgid_plural "%d differences found"
msgstr[0] "%d rozdíl nalezen"
msgstr[1] "%d rozdíly nalezeny"
msgstr[2] "%d rozdílů nalezeno"

#: ../duplicity/dup_main.py:898
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Nenalezeny žádné zbytečné soubory, během úklidu nebylo nic smazáno."

#: ../duplicity/dup_main.py:903
msgid "Deleting this file from backend:"
msgid_plural "Deleting these files from backend:"
msgstr[0] "Mazání tohoto souboru z podpůrné vrtvy (backend):"
msgstr[1] "Mazání těchto souborů z podpůrné vrtvy (backend):"
msgstr[2] "Mazání těchto souborů z podpůrné vrtvy (backend):"

#: ../duplicity/dup_main.py:914
msgid "Found the following file to delete:"
msgid_plural "Found the following files to delete:"
msgstr[0] "Nalezen následující soubor ke smazání:"
msgstr[1] "Nalezeny následující soubory ke smazání:"
msgstr[2] "Nalezeno následujících souborů ke smazání:"

#: ../duplicity/dup_main.py:917
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Spusť duplicity znovu s volbou --force pro skutečné mazání."

#: ../duplicity/dup_main.py:960
msgid "There are backup set(s) at time(s):"
msgstr "Jsou zde sady záloh v časech:"

#: ../duplicity/dup_main.py:962
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Které není možné smazat protože na nich závisí novější sady."

#: ../duplicity/dup_main.py:966
msgid ""
"Current active backup chain is older than specified time.  However, it will "
"not be deleted.  To remove all your backups, manually purge the repository."
msgstr ""
"Stávající aktivní řetěz záloh je starší než zadaný čas. Nicméně, nebude "
"smazán. Pro smazání všech svých záloh vyčistěte repozitář ručně."

#: ../duplicity/dup_main.py:979
msgid "No old backup sets found, nothing deleted."
msgstr "Nenalezeny žádné staré sady záloh, nic nebylo smazáno."

#: ../duplicity/dup_main.py:982
msgid "Deleting backup chain at time:"
msgid_plural "Deleting backup chains at times:"
msgstr[0] "Maže se řetěz zálohy v čase:"
msgstr[1] "Mažou se řetězce zálohy v čase:"
msgstr[2] "Maže se řetězců zálohy v čase:"

#: ../duplicity/dup_main.py:994
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Mažou se všechny přírůstkové signatury řetězců počínající v %s"

#: ../duplicity/dup_main.py:996
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "Mažou se všechny řetězy přírůstkové zálohy počínaje v %s"

#: ../duplicity/dup_main.py:999
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Maže se celý řetěz podpisů %s"

#: ../duplicity/dup_main.py:1001
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Maže se celý řetěz záloh %s"

#: ../duplicity/dup_main.py:1007
msgid "Found old backup chain at the following time:"
msgid_plural "Found old backup chains at the following times:"
msgstr[0] "Nalezen starý řetěz záloh v následujících časech:"
msgstr[1] "Nalezeny staré řetězy záloh v následujících časech:"
msgstr[2] "Nalezeno starých řetězů záloh v následujících časech:"

#: ../duplicity/dup_main.py:1011
msgid "Rerun command with --force option to actually delete."
msgstr ""
"Aby došlo ke skutečnému smazání, spusťte příkaz znovu s volbou --force."

#: ../duplicity/dup_main.py:1034
msgid "No old backup sets found."
msgstr "Nebyly nalezeny žádné staré sady záloh."

#: ../duplicity/dup_main.py:1049
#, python-format
msgid "Signature %s already replicated"
msgstr "Podpis %s je už replikovaný"

#: ../duplicity/dup_main.py:1056 ../duplicity/dup_main.py:1095
#, python-format
msgid "Replicating %s."
msgstr "Replikuje se %s."

#: ../duplicity/dup_main.py:1083
#, python-format
msgid "Backupset %s already replicated"
msgstr "Sada záloh %s je už replikovaná"

#: ../duplicity/dup_main.py:1231
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Mazání místní %s (neautoritativní z pohledu podpůrné vrstvy)."

#: ../duplicity/dup_main.py:1236
#, python-format
msgid "Unable to delete %s: %s"
msgstr "%s se nedaří smazat: %s"

#: ../duplicity/dup_main.py:1270 ../duplicity/dup_temp.py:271
#, python-format
msgid "Failed to read %s: %s"
msgstr "Nepodařilo se načíst %s: %s"

#: ../duplicity/dup_main.py:1284
#, python-format
msgid "Copying %s to local cache."
msgstr "Kopíruje se %s do mezipaměti na tomto stroji."

#: ../duplicity/dup_main.py:1332
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Místní a vzdálená metadata jsou synchronní, není třeba synchronizace."

#: ../duplicity/dup_main.py:1337
msgid "Synchronizing remote metadata to local cache..."
msgstr "Synchronizace vzdálených metadat do místní mezipaměti…"

#: ../duplicity/dup_main.py:1350
msgid "Sync would copy the following from remote to local:"
msgstr "Sync by zkopíroval následující ze vzdáleného na místní:"

#: ../duplicity/dup_main.py:1353
msgid "Sync would remove the following spurious local files:"
msgstr "Sync by odebralo následující zbytečné místní soubory:"

#: ../duplicity/dup_main.py:1397
msgid "Unable to get free space on temp."
msgstr "Nedaří se zjistit volné místo na dočasném."

#: ../duplicity/dup_main.py:1405
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "Na dočasném prostoru je k dispozici %d, záloha potřebuje přibližně %d."

#: ../duplicity/dup_main.py:1408
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "Na dočasném prostoru je k dispozici %d, záloha využije přibližně %d."

#: ../duplicity/dup_main.py:1416
msgid "Unable to get max open files."
msgstr "Nedaří se zjistit limit počtu otevřených souborů."

#: ../duplicity/dup_main.py:1420
#, python-format
msgid ""
"Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""
"Limit naráz otevřených souborů %s je příliš nízký, měl by být >= 1024.\n"
"Napravte to spuštěním příkazu „ulimit -n 1024“ (nebo s vyšší hodnotou).\n"

#: ../duplicity/dup_main.py:1472
msgid ""
"RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""
"RESTART: První svazek se nepodařilo nahrát před ukončením.\n"
"         Restart není možný, záloha se spouští znovu od začátku."

#: ../duplicity/dup_main.py:1478
#, python-format
msgid ""
"RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr ""
"RESTART: Svazky %d až %d se nepodařilo nahrát před ukončením.\n"
"         Restartuje se záloha u svazku %d."

#: ../duplicity/dup_main.py:1485
#, python-format
msgid ""
"RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last "
"partial\n"
"         backup then restart the backup from the beginning."
msgstr ""
"RESTART: Nemožný stav zálohy: manifest má %d svazků, vzdálený má %d svazků.\n"
"         Restart není monžný. Dduplicity vyčistí poslední částečnou zálohu a "
"pak\n"
"         restartuje zálohu od začátku."

#: ../duplicity/dup_main.py:1506
msgid ""
"\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n"
"\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""
"\n"
"PYTHONOPTIMIZE v prostředí způsobí, že duplicity nerozpozná\n"
"své vlastní zálohy. Odstraňte PYTHONOPTIMIZE z prostředí\n"
"a spusťte zálohu znovu.\n"
"\n"
"Viz https://bugs.launchpad.net/duplicity/+bug/931175\n"

#: ../duplicity/dup_main.py:1529
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Získává se zamykací soubor %s"

#: ../duplicity/dup_main.py:1587
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr "Poslední %s záloha zanechala částečnou sadu, restartuje se."

#: ../duplicity/dup_main.py:1591
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr "Čištění předchozí částečné %s sady záloh, restartuje se."

#: ../duplicity/dup_main.py:1603
msgid "Last full backup date:"
msgstr "Datum poslední úplné zálohy:"

#: ../duplicity/dup_main.py:1605
msgid "Last full backup date: none"
msgstr "Datum poslední plné zálohy: žádná"

#: ../duplicity/dup_main.py:1608
msgid "Last full backup is too old, forcing full backup"
msgstr "Poslední plná záloha je příliš stará, vynucuje se pořízení plné zálohy"

#: ../duplicity/dup_main.py:1657
msgid ""
"When using symmetric encryption, the signing passphrase must equal the "
"encryption passphrase."
msgstr ""
"Při použití symetrického šifrování je třeba, aby podpisovací heslová fráze "
"byla stejná, jako ta šifrovací."

#: ../duplicity/dup_time.py:63
#, python-format
msgid ""
"Bad interval string \"%s\"\n"
"\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Chybný řetězec intervalu „%s“"

#: ../duplicity/dup_time.py:69
#, python-format
msgid ""
"Bad time string \"%s\"\n"
"\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Chybný časový řetězec „%s“"

#: ../duplicity/gpginterface.py:238
msgid "Threading not available -- zombie processes may appear"
msgstr ""
"Zpracování ve vláknech není k dispozici – mohou se objevovat zombie procesy"

#: ../duplicity/gpginterface.py:705
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr "GPG proces %d ukončen před funkcí wait()"

#: ../duplicity/lazy.py:352
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Varování: staryidnex %s >= novyindex %s"

#: ../duplicity/lazy.py:427
#, python-format
msgid "Error '%s' processing %s"
msgstr "Chyba „%s“ při zpracování %s"

#: ../duplicity/lazy.py:437
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Přeskakuje se %s kvůli předchozí chybě"

#: ../duplicity/librsync.py:195
msgid ""
"basis_file must be a (true) file or an object whose file attribute is the "
"underlying true file object"
msgstr ""
"je třeba, aby basis_file byl (skutečný) soubor nebo objekt, jehož souborový "
"atribut je je skutečným souborovým objektem"

#: ../duplicity/manifest.py:96
#, python-format
msgid ""
"Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr ""
"Fatální chyba: Stroj se zdrojem zálohy se změnil:\n"
"Stávající stroj: %s\n"
"Předchozí stroj: %s"

#: ../duplicity/manifest.py:103
#, python-format
msgid ""
"Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr ""
"Fatální chyba: Složka se zdrojem záloh se změnila.\n"
"Stávající složka: %s\n"
"Předchozí složka: %s"

#: ../duplicity/manifest.py:113
msgid ""
"Aborting because you may have accidentally tried to backup two different "
"data sets to the same remote location, or using the same archive directory.  "
"If this is not a mistake, use the --allow-source-mismatch switch to avoid "
"seeing this message"
msgstr ""
"Přerušuje se protože byste si mohli nedopatřením pokusit zazálohovat dvě "
"různé datové sady do stejného vzdáleného umístění nebo použít stejnou "
"archivní složku. Pokud to není chyba, použijte přepínač --allow-source-"
"mismatch, aby se toto hlášení neobjevovalo."

#: ../duplicity/manifest.py:215
#, python-format
msgid "Found manifest volume %s"
msgstr "Nalezen svazek manifestu %s"

#: ../duplicity/manifest.py:222
#, python-format
msgid "Found %s volumes in manifest"
msgstr "Nalezeno %s svazků v manifestu"

#: ../duplicity/manifest.py:239
#, python-format
msgid ""
"Manifest file '%s' is corrupt: File count says %d, File list contains %d"
msgstr ""
"Soubor s manifestem „%s“ je poškozený: Počet souborů uvádí %d, seznam "
"souborů obsahuje %d"

#: ../duplicity/manifest.py:258
msgid "Manifests not equal because different volume numbers"
msgstr "Manifesty se neshodují kvůli rozdílným počtům svazků"

#: ../duplicity/manifest.py:263
msgid "Manifests not equal because volume lists differ"
msgstr "Manifest není stejný protože se liší seznamy svazků"

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because hosts or directories differ"
msgstr "Manifesty se neshodují protože stroje nebo složky se liší"

#: ../duplicity/manifest.py:425
msgid "Warning, found extra Volume identifier"
msgstr "Varování, nalezeny extra identifikátory svazků"

#: ../duplicity/manifest.py:451
msgid "Other is not VolumeInfo"
msgstr "Druhý není VolumeInfo"

#: ../duplicity/manifest.py:454
msgid "Volume numbers don't match"
msgstr "Čísla svazků se neshodují"

#: ../duplicity/manifest.py:457
msgid "start_indicies don't match"
msgstr "Start_indicies se neshodují"

#: ../duplicity/manifest.py:460
msgid "end_index don't match"
msgstr "end_index se neshoduje"

#: ../duplicity/manifest.py:467
msgid "Hashes don't match"
msgstr "Otisky (hash) se neshodují"

#: ../duplicity/patchdir.py:84 ../duplicity/patchdir.py:89
#, python-format
msgid "Patching %s"
msgstr "Opravuje se %s"

#: ../duplicity/patchdir.py:550
#, python-format
msgid "Error '%s' patching %s"
msgstr "Chyba „%s“ při opravování %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "Zápisování %s typu %s"

#: ../duplicity/path.py:116
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr ""
"Varování: %s neplatných čísel zařízení (0x%X), nakládáno s tím jako s (0, 0)."

#: ../duplicity/path.py:245 ../duplicity/path.py:306
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Varování: %s má záporný mtime, zacházeno s tím jako s 0."

#: ../duplicity/path.py:370
msgid "Difference found:"
msgstr "Nalezený rozdíl:"

#: ../duplicity/path.py:379
#, python-format
msgid "New file %s"
msgstr "Nový soubor %s"

#: ../duplicity/path.py:382
#, python-format
msgid "File %s is missing"
msgstr "Soubor %s chybí"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Soubor %%s má typ %s, očekáváno %s"

#: ../duplicity/path.py:391 ../duplicity/path.py:417
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Soubor %%s má práva %s, očekáváno %s"

#: ../duplicity/path.py:396
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Soubor %%s má mtime %s, očekáván %s"

#: ../duplicity/path.py:404
#, python-format
msgid "Data for file %s is different"
msgstr "Data pro soubor %s jsou odlišná"

#: ../duplicity/path.py:412
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "symbolický odkaz %%s odkazuje na %s, ale očekáváno %s"

#: ../duplicity/path.py:421
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Soubor zařízení %%s má čísla %s, očekáváno %s"

#: ../duplicity/path.py:603
#, python-format
msgid "Making directory %s"
msgstr "Vytváření složky %s"

#: ../duplicity/path.py:613
#, python-format
msgid "Deleting %s"
msgstr "Mazání %s"

#: ../duplicity/path.py:622
#, python-format
msgid "Touching %s"
msgstr ""
"Operace touch (pokud neexistuje vytvořit; pokud ano, změnit čas posledního "
"přístupu) na %s"

#: ../duplicity/path.py:629
#, python-format
msgid "Deleting tree %s"
msgstr "Mazání stromu %s"

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr "Chyba při vypisování obsahu složky %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Skipping socket %s"
msgstr "Vynechává se soket %s"

#: ../duplicity/selection.py:125
#, python-format
msgid "Error initializing file %s"
msgstr "Chyba při inicializaci souboru %s"

#: ../duplicity/selection.py:129 ../duplicity/selection.py:154
#: ../duplicity/selection.py:454
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Chyba při přístupu k nejspíš zamčenému souboru %s"

#: ../duplicity/selection.py:169
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Varování: základ %s neexistuje, pokračuje se"

#: ../duplicity/selection.py:172 ../duplicity/selection.py:190
#: ../duplicity/selection.py:193
#, python-format
msgid "Selecting %s"
msgstr "Vybírá se %s"

#: ../duplicity/selection.py:290
#, python-format
msgid ""
"Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr ""
"Fatální chyba: specifikace souboru\n"
"    %s\n"
"neodpovídá žádnému ze souborů v základní složce\n"
"    %s\n"
"Použitelné specifikace souboru začínají základní složkou nebo  nějakým\n"
"vzorem (jako např. „**“) což se shoduje se základní složkou."

#: ../duplicity/selection.py:299
#, python-format
msgid ""
"Fatal Error while processing expression\n"
"%s"
msgstr ""
"Fatální chyba při zpracovávání výrazu\n"
"%s"

#: ../duplicity/selection.py:308
#, python-format
msgid ""
"Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr ""
"Poslední výběrový dotaz:\n"
"    %s\n"
"určuje pouze soubory, které zahrnout. Protože výchozí je\n"
"zahrnout všechny soubory, výraz je nadbytečný. Ukončuje se,\n"
"protože to nejspíš není to, co chcete."

#: ../duplicity/selection.py:364
#, python-format
msgid "Reading globbing filelist %s"
msgstr "Čtení seznamu souborů %s, určeného zástupnými znaky"

#: ../duplicity/selection.py:397
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Chyba při kompilaci regulárního výrazu %s"

#: ../duplicity/tempdir.py:160
#, python-format
msgid "Using temporary directory %s"
msgstr "S použitím dočasné složky %s"

#: ../duplicity/tempdir.py:204
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Registruje se (mktemp) dočasný soubor %s"

#: ../duplicity/tempdir.py:226
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Registruje se (mkstemp) dočasný soubor %s"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Zahazování dočasného souboru %s"

#: ../duplicity/tempdir.py:261
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr ""
"Pokus o zapomenutí neznámého dočasného souboru %s – toto je pravděpodobně "
"chyba v programu."

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Odebírání pořád zapamatovaného dočasného souboru %s"

#: ../duplicity/tempdir.py:283
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Vyčištění dočasného souboru %s se nezdařilo"

#: ../duplicity/tempdir.py:288
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr ""
"Čištění dočasné složky %s se nezdařilo – toto je nejspíš chyba v programu."

#: ../duplicity/util.py:144
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "IGNORED_ERROR: Varování: ignoruje se chyba, jak požadováno: %s: %s"

#: ../duplicity/util.py:211
#, python-format
msgid "Releasing lockfile %s"
msgstr "Uvolňování souboru zámku %s"

#~ msgid ""
#~ "For certificate verification with python 2.7.8 or earlier a cacert "
#~ "database\n"
#~ "file is needed in one of these locations: %s\n"
#~ "Hints:\n"
#~ "  Consult the man page, chapter 'SSL Certificate Verification'.\n"
#~ "  Consider using the options --ssl-cacert-file, --ssl-no-check-"
#~ "certificate ."
#~ msgstr ""
#~ "Pro ověřování certifikátů s python 2.7.8 a staršími je třeba souboru\n"
#~ "s databází certifikátů cert. autorit v jednom z těchto umístění: %s\n"
#~ "Rady:\n"
#~ "  Konzultujte manuálovou stránku, kapitolu „Ověřování SSL certifikátů“.\n"
#~ "  Zvažte použití voleb --ssl-cacert-file, --ssl-no-check-certificate ."

#~ msgid ""
#~ "Warning: exclude-device-files is not the first selector.\n"
#~ "This may not be what you intended"
#~ msgstr ""
#~ "Varování: vynechání souborů zařízení není první výběr.\n"
#~ "To nemusí být to, co chcete"

#~ msgid "MultiBackend: list from %s: %s"
#~ msgstr "MultiBackend: seznam z %s: %s"
